exercises/rational-numbers/canonical-data.json

```
@@ -287,6 +287,26 @@
           },
           "expected": [-1, 8]
         },
+        {
+          "uuid": "c291cfae-cfd8-44f5-aa6c-b175c148a492",
+          "description": "Raise a positive rational number to a negative integer power",
+          "property": "exprational",
+          "input": {
+            "r": [3, 5],
+            "n": -2
+          },
+          "expected": [25, 9]
+        },
+        {
+          "uuid": "45cb3288-4ae4-4465-9ae5-c129de4fac8e",
+          "description": "Raise a negative rational number to a negative integer power",
+          "property": "exprational",
+          "input": {
+            "r": [-3, 5],
+            "n": -2
+          },
+          "expected": [-25, 9]
```

as I understand it, the result should be positive here, because a negative times a negative (we square the number) will be positive.

However, if we had taken the power -3, it would remain negative.

---

```suggestion
          "expected": [25, 9]
```

We could have cases for both `n = -2` and `n = -3` too.

---

D'oh :facepalm: you are right!  Fixed the test and added a new one with an odd negative power.